task/sast-coverity-check/0.3/README.md

```diff
@@ -38,8 +41,7 @@ The characteristics of these tasks are:
 
 // TODO: Add reference to private repo for the container image once the task is migrated to repo
 
-
 ## Additional links:
 
-* https://sig-product-docs.synopsys.com/bundle/coverity-docs/page/commands/topics/coverity_capture.html
-* https://sig-product-docs.synopsys.com/bundle/coverity-docs/page/cli/topics/options_reference.html
+* <https://sig-product-docs.synopsys.com/bundle/coverity-docs/page/commands/topics/coverity_capture.html>
```

Comment:
Why `<>` is added here? I can't see the difference when I read https://github.com/konflux-ci/build-definitions/tree/sast-kfp-note/task/sast-coverity-check/0.3

Reply:
It was just a linting suggestion from my editor:

https://github.com/DavidAnson/markdownlint/blob/main/doc/md034.md

I can see why it's distracting though. I'll avoid such changes in future PRs.